packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -553,6 +612,50 @@ const Dashboard: FC = () => {
 
     const hasDashboardTiles = dashboardTiles && dashboardTiles.length > 0;
 
+    const currentTabHasTiles = sortedTiles?.some((tile) => {
+        return (
+            tile.tabUuid === activeTab?.uuid ||
+            (!tile.tabUuid && activeTab?.uuid === defaultTabUuid) ||
+            (tile.tabUuid && !tabs.find((t) => t.uuid === tile.tabUuid))
+        );
+    });
```

Comment:
Found hard to follow what scenarios this covers. Would this logic work? 
```suggestion
    const currentTabHasTiles = sortedTiles?.some((tile) => {
       const tileTabUuid = tile.tableUuid ?? defaultTabUuid; // inherit default tab uuid if needed
        return  tileTabUuid === activeTab?.uuid;
    });
```